### ISSUE TITLE
Err: interface conversion: interface {} is map[string]string, not map[string]interface {} 

### ISSUE THREAD

**Issue by flyize** [Threadfin/Threadfin#226]
#### Please first read the [documentation](https://github.com/Threadfin/Threadfin/blob/main/en/configuration.md). Problems affecting the client, e.g. Plex or Emby should be reported there in the forum.

**Describe the bug**

**Server (please complete the following information):**
- OS Unraid
- Arch x64
- Threadfin Version and Build 1.0b24

**To Reproduce**
Start the container
See error

**Expected behavior**
It boots

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**
```
* * * * * FATAL ERROR * * * * *
OS:   linux
Arch: amd64
Err:  interface conversion: interface {} is map[string]string, not map[string]interface {}

panic.go:838 runtime.gopanic
iface.go:262 runtime.panicdottypeE
xepg.go:631 threadfin/src.mapping
xepg.go:111 threadfin/src.buildXEPG
config.go:268 threadfin/src.StartSystem
threadfin.go:197 main.main
proc.go:259 runtime.main
asm_amd64.s:1572 runtime.goexit

* * * * * * * * * * * * * * * *
```
I have deleted every XML I can find and its still crashing.

**Comment by flyize** [comment]
So I started removing the json files and attempting to restart. It crashed every time until I removed the settings file. And even when I restore a settings file from a week ago, when everything was working, it fails.

**Comment by flyize** [comment]
So I totally rebuilt everything. I was able to add my HDHomeRun and XML for it, as well as my IPTV stuff. As soon as I added my first filter for the IPTV stuff, the whole thing crashed. Now I fear I have to set it all back up again fresh.

**Comment by flyize** [comment]
@Fyb3roptik Any ideas here? I'd be happy to share the XML somewhere privately if you're willing to take a look.

**Comment by Fyb3roptik** [comment]
Join the discord and ping me

**Comment by flyize** [comment]
I'm sorry, but which Discord?

**Comment by Fyb3roptik** [comment]
https://discord.com/invite/rWhzhQTM

**Comment by Fyb3roptik** [comment]
Patched in 1.0.25
